### PR TITLE
Adfs Module: Improve Inputs and Outputs for the AdfsAttributeStore Cmdlets

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR improves the descriptions for the `Inputs` and `Outputs` section of the following `AdfsAttributeStore` cmdlets:

- Add-AdfsAttributeStore
- Get-AdfsAttributeStore
- Remove-AdfsAttributeStore

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR whenever you are available. Thanks.

File: docset/windows/adfs/Add-AdfsAttributeStore.md [hunk 1]
```diff
@@ -177,9 +177,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
 ## INPUTS
 
+### None
+
 ## OUTPUTS
 
-### None
+### None or Microsoft.IdentityServer.Management.Resources.AttributeStore
+
+Returns the new AttributeStore object when the PassThru parameter is specified. By default, this cmdlet does not generate any output.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Add-AdfsAttributeStore.md]:
```suggestion
Returns the new AttributeStore object when the *PassThru* parameter is specified. By default, this cmdlet does not generate any output.
```

File: docset/windows/adfs/Remove-AdfsAttributeStore.md [hunk 2]
```diff
@@ -136,10 +136,15 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
 ### Microsoft.IdentityServer.PowerShell.Resources.AttributeStore
-You can pipe a class structure that represents an AD FS 2.0 attribute store object.
+
+An AttributeStore object is received by the *TargetName* parameter.
 
 ## OUTPUTS
 
+### None or Microsoft.IdentityServer.Management.Resources.AttributeStore
+
+Returns the removed AttributeStore object when the PassThru parameter is specified. By default, this cmdlet does not generate any output.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Remove-AdfsAttributeStore.md]:
```suggestion
Returns the removed AttributeStore object when the *PassThru* parameter is specified. By default, this cmdlet does not generate any output.
```

**Pull request merged**